packages/ant-component-mapper/src/files/form-template.js

```diff
@@ -6,7 +6,7 @@ import FormTemplate from '@data-driven-forms/common/src/form-template';
 const { Title, Paragraph } = Typography;
 
 const Form = ({ layout, children, onSubmit, ...props }) => (
-  <AntForm onFinish={onSubmit} layout={layout ? layout : 'vertical'}>
+  <AntForm onFinish={onSubmit} layout={layout ? layout : 'vertical'} {...props}>
```

Comment:
can we make the `layout` either a default prop or assign it in the function arguments?